src/prisma/schema.prisma

```diff
@@ -666,6 +657,7 @@ model User {
   blacklistedUser               BlacklistedUser[]
   proposalEvaluationPermissions ProposalEvaluationPermission[]
   decidedProposalEvaluations    ProposalEvaluation[]
+  userOTP                       UserOTP?
```

Comment:
i would just call this field "otp" everything tied to this user is already user-related

Reply:
Done